index.html

```diff
@@ -46,7 +46,7 @@
       }
```

Comment:
```suggestion
        box-sizing: border-box;
      }
```

Reply:
What kind of vertical overflow do you observe?

Reply:
It's super minor, we can probably ignore it. Just when I open the page, it seems to exceed the viewport size and creates scrollbars. But maybe it's coming from somewhere else, to be honest, I'm not used to CSS-ing without resets 😅 

<img width="1465" alt="Screenshot 2025-02-21 at 12 08 17" src="https://github.com/user-attachments/assets/08524c47-ef60-4998-b4fc-c1e7f74c6544" />